sunpy/map/mapbase.py

```diff
@@ -2687,7 +2687,7 @@ def contour(self, level, **kwargs):
         """
         Returns coordinates of the contours for a given level value.
 
-        For details of the contouring algorithm see `skimage.measure.find_contours`.
+        For details of the contouring algorithm, see `contourpy.contour_generator`.
```

Comment:
The library will need to be added to the intersphinx dict in conf.py for this to work.

Reply:
I was unaware of that, thanks